README.md

```diff
@@ -143,7 +143,20 @@ For PRs that just modify code in the Formplayer repo, submit a PR to Formplayer
 
 Formplayer also has a dependency on the commcare-core repository. The commcare-core `master` branch is not
 stable and Formplayer uses a different branch. The submodule repo `libs/commcare` should always be pointing to
-the `formplayer` branch.
+the HEAD of the `formplayer` branch.
+
+When making changes to Formplayer that depend on changes to commcare-core you should follow this process:
+
+1. Make changes to commcare-core and push the changes
+2. Make changes to formplayer and update the commcare-core submodule ref to point to the HEAD commit of your
```

Comment:
We can't do this since it results in unresolvable staging conflicts due to core pointing to 2 different heads. Today, I leave the core pointing to `formplayer` and verify the tests locally.